### DISCUSSION TITLE
add_argument in NNI

### DISCUSSION THREAD

**Discussion by Dannypung** [microsoft/nni#3747] [Q&A] [answered]
**Describe the issue**:

parser.add_argument('--tolerance',type=int,default=100,help='tolerance for earlystopping')

If my parameters are set this way, can I search for better parameters directly as shown in the example ？If not, how do I change the code？THX

**Comment by ultmaster** [comment, marked as answer]
Yes. Please follow the example. You can overwrite the args with the parameters got from `nni.get_next_parameter()`.
